hokusai/production.yml

```diff
@@ -74,7 +74,11 @@ metadata:
     service.beta.kubernetes.io/aws-load-balancer-ssl-ports: '443'
     service.beta.kubernetes.io/aws-load-balancer-cross-zone-load-balancing-enabled: 'true'
     service.beta.kubernetes.io/aws-load-balancer-connection-draining-enabled: 'true'
-    service.beta.kubernetes.io/aws-load-balancer-connection-idle-timeout: '300'
+    service.beta.kubernetes.io/aws-load-balancer-connection-idle-timeout: '5'
+    service.beta.kubernetes.io/aws-load-balancer-access-log-enabled: 'true'
+    service.beta.kubernetes.io/aws-load-balancer-access-log-emit-interval: '60'
+    service.beta.kubernetes.io/aws-load-balancer-access-log-s3-bucket-name: 'artsy-elb-logs'
+    service.beta.kubernetes.io/aws-load-balancer-access-log-s3-bucket-prefix: 'production-force'
```

Comment:
Is there some existing data present in the configuration file to programatically derived this term?

Reply:
No - you have to check the docs for our k8s version - so for 1.10 https://v1-10.docs.kubernetes.io/docs/concepts/cluster-administration/cloud-providers/#aws